tin-can-phone.ino

```diff
@@ -196,7 +197,8 @@ class Can {
 
         void openWebSocket(String ip, int port) {
             this->webSocket.beginSocketIO(ip, port);
-            //webSocket.setAuthorization("user", "Password"); // HTTP Basic Authorization
+            String header_string = String("name:") + TIN_CAN_ID;
+            this->webSocket.setExtraHeaders(header_string.c_str());
             this->webSocket.onEvent(webSocketEvent);
         }
 
```

Comment:
On line 269, below here, you should build the URL with the constants defined in the configuration file.

Reply:
(You can't see the line I'm referring to in this snippet, by the way. For some reason I couldn't comment on a part of the code that hasn't changed.)